IDEOorg/steps
# #161396434 Fix template task edit button

#### What does this PR do?
When right clicked and `open link in new tab` is selected, make the template task edit button open the edit form on a new tab

#### Description of Task to be completed?
- install dependencies
- fix the template task edit button

#### How should this be manually tested?
- Right-click the template task edit button and open link on a new tab
- You should see the edit form on a new tab

#### Any background context you want to provide?
Prior to this pull request, when the edit button is right clicked and `open link in new tab` is selected, a blank screen is seen instead of the edit task form.

#### What are the relevant pivotal tracker stories?
#161396434

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
